### PR TITLE
🌐 Add Portuguese translation for `docs/pt/docs/tutorial/extra-models.md`

### PR DESCRIPTION
**Pull request by LorhanSohaky**

### CODE REVIEW

File: docs/pt/docs/tutorial/extra-models.md [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+
+    ```Python hl_lines="1  14-15  18-20  33"
+    {!> ../../../docs_src/extra_models/tutorial003_py310.py!}
+    ```
+
+### `Union` no Python 3.10
+
+Neste exemplo, passamos `Union[PlaneItem, CarItem]` como o valor do argumento `response_model`.
+
+Porque estamos passando-o como um **valor para um argumento** em vez de colocá-lo em uma **anotação de tipo**, precisamos usar `Union` mesmo no Python 3.10.
```
Comment (fabioueno) [inline docs/pt/docs/tutorial/extra-models.md:200]: Small suggestion:
```suggestion
Como estamos passando-o como um **valor para um argumento** em vez de colocá-lo em uma **anotação de tipo**, precisamos usar `Union` mesmo no Python 3.10.
```

Or

```suggestion
Dado que estamos passando-o como um **valor para um argumento** em vez de colocá-lo em uma **anotação de tipo**, precisamos usar `Union` mesmo no Python 3.10.
```

**Review by fabioueno — COMMENTED** [review]
Nice translations!!

This is my first review here, I hope I'm not breaking any rules. 😶

File: docs/pt/docs/tutorial/extra-models.md [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+    username = user_dict["username"],
+    password = user_dict["password"],
+    email = user_dict["email"],
+    full_name = user_dict["full_name"],
+    hashed_password = hashed_password,
+)
+```
+
+!!! warning
+    As funções adicionais de suporte são apenas para demonstração de um fluxo possível dos dados, mas é claro que elas não fornecem segurança real. E é importante seguir as boas práticas de segurança, como criptografar senhas antes de armazená-las em banco de dados, e não armazenar senhas em texto plano.
```
[the remark this answers is not in the document]
Comment (LorhanSohaky) [reply docs/pt/docs/tutorial/extra-models.md]: I fixed the error. It had been the autocomplete

**Review by fabioueno — APPROVED** [review]
Looking good! 🤩

**Review by oandersonmagalhaes — APPROVED** [review]
LGTM 🚀

**Comment by tiangolo** [comment]
Thank you @LorhanSohaky! :man_dancing: :tada: 

And thanks for the reviews @fabioueno and @oandersonmagalhaes :clap: :coffee:

**Pull request merged**